composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "shopware/core": "~6.6.0"
+        "shopware/core": "^6.6.0"
```

Comment:
is it compatible to 6.8.0? 
Otherwise, please use `~6.6.0||~6.7.0`

Reply:
Changed to `~6.6.0||~6.7.0`